.github/workflows/ci.yml

```diff
@@ -11,6 +11,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
+      - name: Check if it is a dependency that should be managed by Expo
+        if: github.actor == 'dependabot[bot]'
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const script = require(
+              `${process.env.GITHUB_WORKSPACE}/scripts/check-expo-dependencies.js`
```

Comment:
I don't think prefixing this with the workspace env var is necessary. the working folder is already that where the code is checked out